proto/public/api.proto

```diff
@@ -124,8 +124,62 @@ message ApiError {
   string error = 1;
 }
 
+message ResourceSpec {
+  string namespace = 1;
+  string resource = 2;
+}
+
+message ResourceSelection {
+  ResourceSpec spec = 1;
+  string label_selector = 2;
+}
+
+message ResourceNotFound {
+  ResourceSpec spec = 1;
+  ApiError error = 2;
+}
+
+message StatSummaryRequest {
+  ResourceSelection resource = 1;
+  TimeWindow time_window = 2;
+
+  oneof outbound {
+    Empty none = 3;
+    ResourceSpec out_to_resource   = 4;
+    ResourceSpec out_from_resource = 5;
+  }
+}
+
+message StatSummaryResponse {
+  repeated StatTable stat_tables = 1;
+  ResourceNotFound error = 2;
+}
+
+message StatTable {
+  oneof table {
+    PodGroup pod_group = 1;
+  }
+
+  message PodGroup {
+    repeated Row rows = 1;
+
+    message Row {
+      ResourceSpec spec = 1;
+
+      uint64 meshed_pod_count = 2;
+      uint64 total_pod_count = 3;
+
+      double volume_per_sec = 4;
+      double success_rate = 5;
```

Comment:
tioli:

perhaps it makes sense to expose these as:
```
     uint64 success_volume = 4;
     uint64 failure_volume = 5;
```

rps is calculated by adding these together and dividing by the number of seconds in the window.

success rate is calculated by dividing success_volume by the sum of the two volumes.

Reply:
hmmmm, yeah, we can definitely do that. I was thinking about how we currently do these queries using `irate` from prometheus directly, but since we're not using this to get timeseries, it does make sense that we can just return the total, and calculate the rate ourselves.

Reply:
and `_count` is probably a more obvious suffix than `_volume`...